Unity LTS & porting guides update (#2326)
MicrosoftDocs/mixed-reality: mixed-reality-docs/mixed-reality-cloud-services.md

---
layout: LandingPage
title: Mixed reality cloud services
description: Mixed reality cloud services resources.
author: hferrone
ms.author: v-haferr
ms.date: 06/5/2020
ms.topic: overview
ms.localizationpriority: high
keywords: Mixed Reality, develop, development, HoloLens, cloud services
---

# Azure Cloud Services

## Overview

> Azure Cloud Services can now go hand-in-hand with Mixed Reality on a variety of platforms with Azure Remote Rendering and Azure Spatial Anchors. If you're developing in Unity, you also have access to a wide array of add-on cloud APIs that can be easily integrated into your projects.

Reviewer: Would start this off talking about Mixed Reality Services specifically, first - i.e. how MR Services such as Azure Remote Rendering and Azure Spatial Anchors enable developers to build compelling MR solutions. And then would do the transition into 'hey and there are also other Azure Services that you can integrate into your projects etc etc (i.e. the ones at the bottom of this article) 

Some 'MR Services' wording from our ACOM page (https://azure.microsoft.com/en-us/topic/mixed-reality/) that could be a good reference point: 

_Intelligent services
Make the world your app canvas. Build applications with spatial awareness to enable 3D training, predictive equipment maintenance, design review, and more in the context of your users’ environments._

Author: Fixed!

## Mixed Reality services

### Azure Remote Rendering

> [Azure Remote Rendering](https://docs.microsoft.com/azure/remote-rendering) (ARR) is a service that lets you to render highly complex 3D models in real time. ARR is currently in public preview and can be added to your Unity or Native C++ projects  if you're targeting HoloLens 2 or Windows desktop PC.

Reviewer: @rapete any feedback/wording tweaks from you, Jonathan, or any of the other ARR crew on wording?

Reviewer 2: ```suggestion
[Azure Remote Rendering](https://docs.microsoft.com/azure/remote-rendering) (ARR) is a service that lets you to render highly complex 3D models in real time. ARR is currently in public preview. It can be added to your Unity or Native C++ projects targeting HoloLens 2 or Windows desktop PC.
```

Author: Fixed!

### Azure Spatial Anchors

> [Azure Spatial Anchors](https://docs.microsoft.com/azure/spatial-anchors) (ASA) is a cross-platform service that allows you to create mixed reality experiences using objects that persist their location across devices. The service can be developed in a host of environments and deployed to a large group of devices and platforms. This gives them special dispensation for their own list of available platforms:

Reviewer: Wording suggestion (specifically the sentence about what ASA is): 

[Azure Spatial Anchors](https://docs.microsoft.com/azure/spatial-anchors) (ASA) is a cross-platform service that allows you to build spatially aware mixed reality applications. With Azure Spatial Anchors, you can map, persist, and share holographic content across multiple devices, at real-world scale. The service can be developed in a host of environments and deployed to a large group of devices and platforms. This gives them special dispensation for their own list of available platforms:

Author: Fixed!

* Unity for HoloLens
* Unity for iOS
* Unity for Android
* Native iOS
* Native Android
* C++/WinRT and DirectX for HoloLens
* Xamarin for iOS
* Xamarin for Android

## Add-on services

> If you're developing in Unity, check out the tutorials listed below for step-by-step cloud services integration into your new or existing projects.

Reviewer: Would add more context here around, how the services in the list below are additional [non-MR] services that can be integrated into projects being developer in Unity (as it reads right now, it sounds like the services below are add-on's to existing MR services, when really they are all standalone)

Author: Fixed!
